pydm/widgets/image.py

```diff
@@ -287,18 +287,39 @@ def image_width_changed(self, new_width):
             return
         self._image_width = int(new_width)
 
+    @staticmethod
```

Comment:
Is it necessary to be a static method?
I have an application where I reimplemented `redrawImage` to add some processing, but I need some object attributes in this processing. So in this case, for example, I need this `process_image` to be a normal method.

Reply:
Replied on the previous comment.